### DISCUSSION TITLE
🃏 Joker that completes Batman

### DISCUSSION THREAD

**Discussion by hyunjimoon** [Data4DM/BayesSD#71] [people; relating]
## Joker
- hierachical bayesian
- colinearity 

## Batman
- ADC prior
- multimodality

**Comment by tomfid** [comment]
Do you have a link for the ADC prior?

**Reply by hyunjimoon** [reply]
https://betanalpha.github.io/assets/case_studies/hierarchical_modeling.html#43_Now_I_Know_my_ADCs

**Comment by hyunjimoon** [comment]
From Niko's description of monster model, it (unlike planetary model) doesn't suffer from multi modality https://github.com/nsiccha/monster#incremental-and-adaptive-warm-up

**Comment by hyunjimoon** [comment]
We need HMC to prove HMC's value. Another example of Competitively Complementary (e.g. Joker-Batman, 🐣 Chicken-Egg, Prey-Predator, Innovative-Regulative, Frequentist-Bayesian) relation. Two stock negative loop oscillator.


Dynamic MCMC provides better diagnostics, on which it relies to prove its value. Example of non-dynamic MCMC can be accept and reject, gibbs sampling, particle filter (could particle filtering also be gradient-based and geometric i.e. gliding along geodesic?).


Btw, it is interesting competitive prey-predator shares spirit with collaborative chicken-egg. I wish to have discussion on this idea. I gathered examples in the [Guard-Attack table](https://github.com/Data4DM/BayesSD/blob/master/DailyDigest/4%20System%20Structures.md). Re: your question on cookbook basecamp, I think [DailyDigest folder](https://github.com/Data4DM/BayesSD/tree/master/DailyDigest) holds useful logs to heat our cooking, could we put documents there? @tomfid
